src/OnboardingContext.tsx

```diff
@@ -29,23 +32,37 @@ export const OnboardingProvider: FunctionComponent<OnboardingProviderProps> = ({
   children,
   userHasCompletedOnboarding,
 }) => {
-  const [onboardingIsComplete, setOnboardingIsComplete] = useState<boolean>(
+  const [isOnboardingComplete, setIsOnboardingComplete] = useState<boolean>(
     userHasCompletedOnboarding,
   )
+  const [destinationAfterComplete, setDestinationAfterComplete] = useState<
+    Stack
+  >(Stacks.Activation)
 
   const completeOnboarding = () => {
     StorageUtils.setIsOnboardingComplete()
-    setOnboardingIsComplete(true)
+    setIsOnboardingComplete(true)
   }
 
   const resetOnboarding = () => {
     StorageUtils.removeIsOnboardingComplete()
-    setOnboardingIsComplete(false)
+    setDestinationAfterComplete(Stacks.Activation)
```

Comment:
If this method will be called from the contact menu should this have defaulted to these activation screens? Should this argument be passed as an argument if we change this method somewhere else? We could leave the argument as the activation stack by default. Or is this just for debug purposes that we must have the complete flow there?

Reply:
My understanding is that this function will be called from the debug menu and that we want to reset the entire onboarding state to be like it was on first download. So we want to set `isOnboardingComplete` to false and ensure that `destinationAfterComplete` is set to `Stacks.Activation`, because if they have tapped `How the app works` on the Settings screen, `destinationAfterComplete` would be set to `Stacks.Settings`. Does that make sense? I may be misunderstanding you.

Reply:
Roger roger